src/Stateless/Stateless.csproj

```diff
@@ -1,12 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="../../common.props" />
+
   <PropertyGroup>
     <Description>Create state machines and lightweight state machine-based workflows directly in .NET code</Description>
     <AssemblyTitle>Stateless</AssemblyTitle>
     <NeutralLanguage>en-US</NeutralLanguage>
-    <VersionPrefix>4.2.0</VersionPrefix>
     <Authors>Stateless Contributors</Authors>
-    <TargetFrameworks>netstandard1.0;net45;net40</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net47</TargetFrameworks>
```

Comment:
could you please change this with netstandart2.1

Reply:
Why? Currently only .NET Core 3.0 supports .NET Standard 2.1, and that probably won't change. .NET Standard 2.1 is a superset of 2.0, if 2.0 is sufficient I suggest to stay on this version. Supporting .NET Standard 2.0 allows the library to be used on older frameworks.

Reply:
there is no future with .net framework says microsoft (https://devblogs.microsoft.com/dotnet/announcing-net-standard-2-1). so this library must not support older frameworks. i am upgrading all my projects to .net core 3.0 and .net standart 2.1 too. 
and i think this upgrade should done once, because this repo is moving very slowly.

Reply:
Yeah, but supporting it is effortless, and we gain nothing with 2.1. Currently UWP, I think Xamarin, Unity... they all support only .NET Standard 2.0.

Reply:
if you are using EF core 3.0, you will have to upgrade to standart 2.1 like me :)
Anyway, let pr merge with the current version :)

Reply:
Having this library support the lowest common API version doesn't disallow anyone from using a superset in any project, so I agree that unless there is a need for anything from 2.1 within this library, it should stay on 2.0.

Reply:
In fact, my problem is that when I run my current project that compiled with dotnet publish, i am getting an error that cant find an old system.linq file. And I doubt this library because all the others are .netstandard 2.0 or higher.

Reply:
> if you are using EF core 3.0, you will have to upgrade to standart 2.1 like me :)
> Anyway, let pr merge with the current version :)

only if you rely on netstandard 2.1 packages like EFCore 3
netstandard really should be ok, as 2.1 is a small superset

Reply:
net47 not needed as netstandard 2 can be consumed from [net461 and above](https://docs.microsoft.com/en-us/dotnet/standard/net-standard).